Assets/_Project/Scripts/Map/AbstractFakeLocalizationProvider.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Unity.Location;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.Map
+{
+	public abstract class AbstractFakeLocalizationProvider : AbstractLocationProvider
```

Comment:
Do we really need this separation? I'd suggest leaving only ILocationProvider <- FakeLocationProvider, DeviceLocationProvider (unless those two really have something in common, then add custom AbstractLocationProvider with that common functionality).